shell/platform/android/io/flutter/plugin/platform/SingleViewPresentation.java

```diff
@@ -52,7 +52,7 @@
  */
 @Keep
 @TargetApi(Build.VERSION_CODES.JELLY_BEAN_MR1)
-class SingleViewPresentation extends Presentation {
+public class SingleViewPresentation extends Presentation {
```

Comment:
we should look into re-organizing the packages in the embedding. The current structure isn't well suited, and ends up requiring public classes when they aren't intended to be used outside of the embedding internal implementation. cc @xster

Reply:
Sure, we can definitely re-organize things (which is easier if most things stay private to avoid breaking changes). It's low-priority on my mind so far since I don't have many concrete needs for refactoring. What's on your mind?

Reply:
> which is easier if most things stay private to avoid breaking changes

Right.  Although, for the time being we should be able to use https://code.google.com/archive/p/doclava/wikis/JavadocTags.wiki#@hide.